packages/api-clients/react-query/origin-organization-irec-api/package.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "@energyweb/origin-organization-irec-api-react-query-client",
```

Comment:
Should this client libs be available on NPM?

If not, we should put the private flag in their package.json.

Reply:
Why not? Previously we were releasing it on npm

Reply:
We did because we needed them for CD. Do we still need them for deployments? I think now we build all docker images locally and then deploy.

Reply:
But what about other users setting up the origin-ui? 
It is required by Certificate, Organization part of the UI and the root itself